### DISCUSSION TITLE
When analyzing without auto regression

### DISCUSSION THREAD

**Discussion by boozesounds** [ourownstory/neural_prophet#365] [Q&A - get help using NeuralProphet]
I analyzed data that simply repeats 1,2,3,4,5,6,7,8,9,10,9,8,7,6,5,4,3,2,1

When giving lag using auto regression
![3](https://user-images.githubusercontent.com/35713030/126361199-4c124203-c00e-4d0e-aba7-76a0fd792c8c.jpg)
Zoom in on auto regression results
![1](https://user-images.githubusercontent.com/35713030/126360684-a346cd1c-59d2-4d35-98cb-03953d40c164.jpg)

When not to use auto regression

![2](https://user-images.githubusercontent.com/35713030/126360664-78e46a18-ccd8-4366-bcd9-cfaa14cd2348.jpg)

Why is there so much difference? How can I make the analysis work even when autoregression is not used?

**Comment by ourownstory** [comment]
AR can take the current state into account.
time features only can only capture trend and seasonality. you might be able to capture it with a manually tuned seasonality.
